playbooks/roles/common-server/defaults/main.yml

```diff
@@ -52,6 +52,7 @@ COMMON_SERVER_NO_BACKUPS: false
 COMMON_SERVER_OPENCRAFT_REPO: "deb http://debs.opencraft.com/ {{ ansible_distribution_release }} main"
```

Comment:
Sorry I wasn't more explicit, but the idea was to use COMMON_SERVER_OPENCRAFT_REPO_DOMAIN here, too.

Reply:
It has been updated as well !